libs/blocks/media/media.js

```diff
@@ -49,6 +55,47 @@ export default function init(el) {
     if (image) image.classList.add('image');
     const img = image.querySelector(':scope img');
     if (header && img?.alt === '') img.alt = header.textContent;
+
+    // lists
+    if (row.querySelector('ul')) {
+      el.querySelector('ul').classList.add('default-list');
+    }
+
+    // subcopy
+    const actionArea = row.querySelector('p.action-area');
+    if (actionArea?.nextElementSibling?.tagName === 'P') {
+      updateInnerHtml(actionArea.nextElementSibling, 'strong', '');
```

Comment:
By the looks of this you are removing `strong` tags with this function? Is there a reason we can't just update the document to not be bold?

Reply:
Thanks @Sartxi I've updated the docx file & removed updateInnerHtml function since it is not needed anymore https://github.com/adobecom/milo/pull/971/commits/8d5de541e23b41ce57686d6ebd1a62ca71fcfa09